connector.go

```diff
@@ -104,6 +107,63 @@ type Connector struct {
 	requestor      HttpRequestor
 }
 
+func getLatestVersion(hostCfg HostConfig, authCfg AuthConfig, transportCfg TransportConfig) (string, error) {
+	schemaUrl := fmt.Sprintf("https://%s/wapi/v1.0/?_schema", hostCfg.Host)
+	req, err := http.NewRequest(http.MethodGet, schemaUrl, nil)
+	if err != nil {
+		return "", err
+	}
+
+	req.SetBasicAuth(authCfg.Username, authCfg.Password)
+
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: !transportCfg.SslVerify,
+		},
+	}
+
+	client := &http.Client{
+		Timeout:   time.Second * 10,
+		Transport: tr,
+	}
+	resp, err := client.Do(req)
+	if err != nil {
+		return "", err
+	}
+
+	if resp.StatusCode != 200 {
```

Comment:
please use http.StatusOk instead

Reply:
Updated the logic. Now I use the requestor object, which handles incorrect HTTP status codes.